plugins/pubIds/doi/templates/doiSuffixEdit.tpl

```diff
@@ -34,16 +36,23 @@
 					{capture assign=translatedObjectType}{translate key="plugins.pubIds.doi.editor.doiObjectType"|cat:$pubObjectType}{/capture}
 					{capture assign=assignedMessage}{translate key="plugins.pubIds.doi.editor.assigned" pubObjectType=$translatedObjectType}{/capture}
 					<p class="pkp_help">{$assignedMessage}</p>
-					{include file="linkAction/linkAction.tpl" action=$clearPubIdLinkActionDoi contextId="publicIdentifiersForm"}
+					{if !$formDisabled}
+						{include file="linkAction/linkAction.tpl" action=$clearPubIdLinkActionDoi contextId="publicIdentifiersForm"}
+					{/if}
 				</p>
 				{/fbvFormSection}
 			{/if}
 		{else} {* pub id preview *}
 			<p>{$pubIdPlugin->getPubId($pubObject)|escape}</p>
 			{if $canBeAssigned}
-				<p class="pkp_help">{translate key="plugins.pubIds.doi.editor.canBeAssigned"}</p>
-				{assign var=templatePath value=$pubIdPlugin->getTemplateResource('doiAssignCheckBox.tpl')}
-				{include file=$templatePath pubId="" pubObjectType=$pubObjectType}
+				{if !$formDisabled}
+					<p class="pkp_help">{translate key="plugins.pubIds.doi.editor.canBeAssigned"}</p>
+					{assign var=templatePath value=$pubIdPlugin->getTemplateResource('doiAssignCheckBox.tpl')}
+					{include file=$templatePath pubId="" pubObjectType=$pubObjectType}
+				{else}
+					<p class="pkp_help">{translate key="plugins.pubIds.doi.editor.canBeAssigned.disabled"}</p>
+				{/if}
+				
```

Comment:
If the form is disabled, we only want to show a DOI if it is assigned. `$pubIdPlugin->getPubId($pubObject)` will get the DOI that _will be assigned_ when the action is taken. But in such cases, the action can't be taken.

So if the form is disabled show a) the DOI that is assigned to the galley or b) text that says no doi is assigned to the galley.

Reply:
Changed the locale key to `plugins.pubIds.doi.editor.noDoiAssigned` with the text `No DOI is assigned yet.`. Removed the preview of the DOI

Reply:
Can you add a new locale key that says "No DOI is assigned."? The word "yet" implies that someone can change it but once it's published they can't.

Reply:
Changed